app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/Config.scala

```diff
@@ -14,6 +14,8 @@ object Config {
   object server {
     private val serverConfig = config.getConfig("server")
     val timeoutSeconds = serverConfig.getInt("timeoutSeconds")
+    val healthcheckTimeoutSeconds =
+      serverConfig.getInt("healthcheckTimeoutSeconds")
```

Comment:
I gather this is somehow referencing the value in `application.conf`. I'm not entirely sure how these are wired together but I can go with it

Reply:
I can tell you a bit about how config works at some point -- it's a screen share and jump through files kind of thing but in short, there are a few magic file names / locations and some special syntax that tie env vars to application config